src/folding/nova/circuits.rs

```diff
@@ -388,7 +448,7 @@ mod tests {
     /// used as the state. `z_i` is used as `x`, and `z_{i+1}` is used as `y`, and at the next
     /// step, `z_{i+1}` will be assigned to `z_i`, and a new `z+{i+1}` will be computted.
     pub struct TestFCircuit<F: PrimeField> {
-        _f: PhantomData<F>,
+        pub _f: PhantomData<F>,
```

Comment:
Why making this `pub`? It's a test so doesn't matter much. But `ppub(crate)` should do.

Reply:
I definitely thing a `new` function looks cleaner though. But as said, if you prefer it like this, so be it :)

Reply:
Cool, I'll update it adding `new` method :smile_cat: